apps/src/code-studio/components/SublevelCard.jsx

```diff
@@ -8,17 +8,18 @@ import LessonExtrasFlagIcon from '@cdo/apps/templates/progress/LessonExtrasFlagI
 import MazeThumbnail from '@cdo/apps/code-studio/components/lessonExtras/MazeThumbnail';
 import queryString from 'query-string';
 
-const THUMBNAIL_IMAGE_SIZE = 150;
+const THUMBNAIL_IMAGE_SIZE = 200;
 const MARGIN = 10;
+const WIDTH = 435;
 
 const styles = {
   row: {
     display: 'flex',
+    width: WIDTH,
     marginBottom: MARGIN,
     backgroundColor: color.white,
     border: '1px solid rgb(187, 187, 187)',
     borderRadius: 2,
-    width: 400,
     marginRight: 10
```

Comment:
should this be `margin: MARGIN`? How did you decide which places to use the const and which places to leave it as 10?

Reply:
I think only the left and right margins are important to be using MARGIN but thats how we calculate some sizing stuff to make sure the widths of everything is right. I'll double check all the places that should be using it are.

Reply:
Updated